### PR TITLE
Doubt on the md5

### PR DESCRIPTION
**Pull request by SebastianGomezVidal**
Hi,

I did not understand the last point of the GET exercise. I mean, I understand a little bit about cryptography, so my question is to what  do I need to apply a hash algorithm ? since in the response is not any value related to md5, to my understanding this algorithm is obsolete. I am using jscrypto to run hashes' algorithms.

Thanks.

### CODE REVIEW

**Comment by SebastianGomezVidal** [comment]
image

File: test/GithubApi.Repositories.test.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        });
+      });
+
+      describe('Should Get README.md file', () => {
+        it('Should dowload README.md only', async () => {
+          const response = await axios.get(readDownload);
+          console.log(CryptoJS.MD5(response.data));
+
+          expect(response.status).to.equal(StatusCodes.OK);
+          expect(response.statusText).to.equal('OK');
```
Comment (holgiosalos) [inline test/GithubApi.Repositories.test.js:82]: @SebastianGomezVidal to answer your question, the idea is to you add one expect to verify the expected md5 of the response.data which has the content of the readme file. I don't know if crypto-js is the right library for this, there is another library that tells you the md5, called `md5`, but if crypto-js does, you don't need to change it. At the end the expect should look like this:
`expect(md5(response.data)).to.equal("theExpectedMD5Value")`

Comment (SebastianGomezVidal) [reply test/GithubApi.Repositories.test.js:81]: Thanks so much, Holmes. I already implemented the solution.

File: package.json [hunk 2]
```diff
@@ -33,6 +34,7 @@
   },
   "dependencies": {
     "axios": "^0.27.2",
-    "dotenv": "^16.0.1"
+    "dotenv": "^16.0.1",
+    "md5": "^2.3.0"
```
[the remark this answers is not in the document]
Comment (SebastianGomezVidal) [reply package.json]: Sorry about that. I corrected the issue, now it seems good to go

**Review by holgiosalos — APPROVED** [review]

**Pull request merged**